hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpPostCommandProcessor.java

```diff
@@ -44,6 +48,9 @@ public void handle(HttpPostCommand command) {
                 handleManagementCenterUrlChange(command);
             } else if (uri.startsWith(URI_QUEUES)) {
                 handleQueue(command, uri);
+            } else if (uri.startsWith(URI_SHUTDOWN_CLUSTER_URL)) {
+                handleClusterShutdown(command);
+                return;
```

Comment:
Is `return` necessary here? Can't we just set the response of `command` and let the response be sent in line 60.